cuqi/geometry/_geometry.py

```diff
@@ -702,32 +702,60 @@ def _compute_eigpairs(self, xnod, C_nu, sigma, M, N_GL):
 
 class StepExpansion(Continuous1D):
     '''
-    class representation of the step random field with 3 intermidiate steps
+    Class representation of step functions (piecewise constant functions) with `n_steps` 
+    equidistant steps on the interval [0, L].
+    The function `par2fun` maps the parameters `p` (which are the step magnitudes) to the 
+    corresponding step function evaluated on the spacial grid (`grid`) of nodes x0=0, x1, ...xn=L.
+    
+    For example, if `n_steps` is 3 and `grid` is a uniform grid with nodes x0=0, x1=0.1L, ..., xn=L, 
+    then the resulting function evaluated on the grid will be p[0], if x<=L/3, p[1], if L/3<x<=2L/3, 
+    p[2], if 2L/3<x<=L.
+    
+    Parameters:
+    -----------
+    grid: ndarray
+        Grid points for the step expansion to be evaluated at. There can be more grid points 
```

Comment:
There "can" be.. Must there be? The error raised below seems to indicate that the number of grid point MUST be larger than the number of steps.

Reply:
I updated this documentation to clarify that the number of grid points can be equal n_steps to or more.